2001-3000/LIT2388Taamme.xml

```diff
@@ -55,22 +56,59 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
-         <change who="ES" when="2017-03-07"> updated titles abstract keywords relations</change> 
+         <change who="ES" when="2017-03-07"> updated titles abstract keywords relations</change>
          <change who="MV" when="2017-05-10">Updated keyword</change>
       </revisionDesc>
    </teiHeader>
-   <text xml:base="https://betamasaheft.eu/">
-      <body>
+             <text>
+                 <body>
+                   <div type="edition">
+                     <note>The text follows <bibl><ptr target="bm:Turaiev1912Trinity"/><citedRange unit="page">322-324</citedRange></bibl>.</note>
+                        <div type="textpart" subtype="chapter" xml:id="StoryTrynity">
+                          <label>Miracle 1. The promises of the Persons
+                            of the Trinity to the faithful.</label>
+                          <div type="textpart" subtype="chapter" xml:id="Miracle1">
+                          <ab>
+                          <gap reason="omitted" resp="DN"/>ወአሐተ፡ ዕለተ፡ ይቤ፡ አብ፡ አብኒ፡ ዘአክበረኒ፡ በምድረ፡ አነ፡ አክበሮ፡
+                           በመንግሥተ፡ ሰማያት፡ ገጸ፡ አርያም፡ ይመሰል፡ አምሳሊሁ፡ ዘወሀቢኒ፡ መባአ፡ ወስንዳሌ፡ ወወይነ፡ እስከ፡ ይትነሣእ> እመካኑ፡ <gap reason="omitted" resp="DN"/>
+                        </ab>
+                       </div>
+
+                       <div type="textpart" subtype="chapter" xml:id="Miracle2">
+                         <label>Miracle 2. The governor is angry with his troops
+                           who were doing the commemoration of the Trinity.</label>
+                         <ab>
+                         <gap reason="ellipsis" resp="DN"/>
+                        ወሀሎ፡ ፩መኰንን፡ ወአሐተ፡ ዕለተ፡ ውሰተ፡ ሀገረ፡ ፍልስጥኤም፡ አዘዘ፡ ይትጋብኡ፡ ለሠራዊቱ። ወሶበ፡
+                        ርእየ፡ መኰንን፡ ከመ፡ ገብኡ፡ ሠራዊቱ፡ ወይቤ፡ መኰንን፡ ለዘገብረ፡ ተዘካረ፡ አብ፡ ወወልድ፡ ወመንፈስ፡
+                        ቅዱስ፡ ይንበር፡ በብሔር፡ <gap reason="omitted" resp="DN"/>
+                         </ab>
+                       </div>
+
+
+                       <div type="textpart" subtype="chapter" xml:id="Miracle3">
+                          <label>Miracle 3. Demons in the beer.</label>
+                          <ab>
+                            ወሀሎ፡ ፩ብእሲ፡ ወአሐተ፡ ዕለተ፡ ውስተ፡ ሀገረ፡ ፍልስጥኤም፡ ወረሰየ፡
+                             ምዘረ፡ በስመ፡ አብ፡ ወወልድ፡ ወመንፈስ፡ ቅዱስ፡ ለገቢረ፡ ተዝካሮሙ፡ ወመጽኡ፡ ብዙኃን፡ ሰይጣናት፡ ወተመሰሉ፡ ከመ፡ አጽልልት፡
+                                   </ab>
+                       </div>
+                                       </div>
+                                       </div>
+                      <div type="bibliography">
          <listRelation>
             <relation name="ecrm:P129_is_about" active="LIT2388Taamme" passive="PRS10775Trinity"/>
+            <relation name="dcterms:isPartOf" active="LIT2388Taamme" passive="LIT2286ZenaSellase"/>
+              <relation name="lawd:hasAttestation" active="LIT2388Taamme" passive="IVorlov40"/>
+              <relation name="dcterms:isPartOf" active="LIT2388Taamme" passive="LIT2282Trinity" cert="medium"/>
```

Comment:
the encoding is correct, but the cert="medium" information will not be rendered and not be seen by users. Maybe it could be useful to include this information also in paragraphic form in the abstract.

Reply:
Why not in the beer? :) I tried, but on the whole the Miracles of Trinity collections are problematic, someone should write them out, sometime.

Reply:
Dorothea means, you should add a "desc" element to the relation saying in words that it *might* be part (or not)

Reply:
Will this work out?